src/SfxWeb/src/app/views/applications/events/events.component.ts

```diff
@@ -19,10 +19,9 @@ export class EventsComponent extends BaseControllerDirective {
    }
 
    setup() {
-    this.listEventStoreData = [{
-      eventsList: this.data.createApplicationEventList(null),
-      displayName: 'Applications'
-    }];
+    this.listEventStoreData = [
+      this.data.getApplicationEventData(null)
```

Comment:
nit: remove the null, given its an optional param it would be better to just not put the null.